src/util/api.js

```diff
@@ -0,0 +1,37 @@
+const BASE_URL = 'https://bootcamp-api.codeit.kr/api';
+
+async function getAPI(query) {
+  try {
+    const response = await fetch(`${BASE_URL}/${query}`);
+
+    if (!response.ok) {

```

Comment:
```suggestion
    if (!response?.ok) {
```
response 자체가 undefined 일 경우에 대비해주세요!